performance/cinterop/build.gradle.kts

```diff
@@ -4,19 +4,22 @@
  */
 
 import org.jetbrains.kotlin.gradle.plugin.mpp.KotlinNativeTarget
+import org.jetbrains.kotlin.gradle.plugin.mpp.NativeBuildType
 
 plugins {
     id("benchmarking")
 }
 
+val defaultBuildType = NativeBuildType.RELEASE
+
 benchmark {
     applicationName = "Cinterop"
     commonSrcDirs = listOf("../../tools/benchmarks/shared/src", "src/main/kotlin", "../shared/src/main/kotlin")
     jvmSrcDirs = listOf("src/main/kotlin-jvm", "../shared/src/main/kotlin-jvm")
     nativeSrcDirs = listOf("src/main/kotlin-native", "../shared/src/main/kotlin-native/common")
     mingwSrcDirs = listOf("src/main/kotlin-native", "../shared/src/main/kotlin-native/mingw")
     posixSrcDirs = listOf("src/main/kotlin-native", "../shared/src/main/kotlin-native/posix")
-
+    buildType = (findProperty("nativeBuildType") as String?)?.let { NativeBuildType.valueOf(it) } ?: defaultBuildType
```

Comment:
I used debuggable and optimized options next in plugin, I think it's better to work with enum than with string values just because of gradle can't provide us enums.
And it'll be more obvious if there is an error during conversion in build script than if it falls in plugin